src/Build/Evaluation/Conditionals/Scanner.cs

```diff
@@ -325,33 +325,35 @@ private static bool ScanForPropertyExpressionEnd(string expression, int index, o
             {
                 fixed (char* pchar = expression)
                 {
+                    if (expression.Length > 1 && pchar[0] == '(' && char.IsWhiteSpace(pchar[1]) && ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave16_10))
```

Comment:
is the first character in the expression guaranteed to be a '('?

Should this maybe move under the `if (character == '(')?

Something like:
```c#
                        if (character == '(')
                        {
                             if (nestLevel == 0 && index + 1 > expression.Length && pchar[index + 1] == '(' && char.IsWhiteSpace(pchar[1]) && ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave16_10))
                              {
                                    indexResult = 1;
                                     return false;
                              }
                            nestLevel++;
                        }
```

Reply:
Yes